libnmstate/nm/ovs.py

```diff
@@ -165,8 +171,15 @@ def _get_bridge_port_info(port_profile, devices_info):
     port_slave_names = [c.get_interface_name() for c in port_slave_profiles]
 
     if port_slave_names:
-        port_info['name'] = port_slave_names[0]
-        port_info['type'] = OBPortType.SYSTEM
+        iface_slave_name = port_slave_names[0]
+        iface_device = device.get_device_by_name(iface_slave_name)
+        port_type = (
```

Comment:
Ok. Although I consider the current one safer, because there is no chance that in the future a condition is removed or added, missing to set the `port_type`.